resources/ext/ezbake.conf

```diff
@@ -5,9 +5,9 @@ ezbake: {
                # TODO: With AIO packages, ruby-load-path is now the same
                # across all platforms, so we don't need this. It should be
                # moved to a regular config file. See SERVER-331.
-               install: ["echo \\\"os-settings: {\\\"                         > %{buildroot}%{_sysconfdir}/%{realname}/conf.d/os-settings.conf",
-                         "echo \\\"    ruby-load-path: [/opt/puppetlabs/agent/lib/ruby/vendor_ruby]\\\" >> %{buildroot}%{_sysconfdir}/%{realname}/conf.d/os-settings.conf",
-                         "echo \\\"}\\\"                                      >> %{buildroot}%{_sysconfdir}/%{realname}/conf.d/os-settings.conf" ],
+               install: ["echo \\\"os-settings: {\\\"                         > ${DESTDIR}${projconfdir}/conf.d/os-settings.conf",
+                         "echo \\\"    ruby-load-path: [/opt/puppetlabs/agent/lib/ruby/vendor_ruby]\\\" >> ${DESTDIR}${projconfdir}/conf.d/os-settings.conf",
```

Comment:
Is "agent" still right here?  The new canonical source - https://github.com/puppetlabs/puppet-specifications/blob/master/file_paths.md - seems to indicate that it should be `/opt/puppetlabs/puppet/lib/ruby/vendor_ruby`.